packages/badge/src/badge.css

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import './spectrum-badge.css';
+
+/* center align icons and text */
+
+:host {
+    display: inline-flex;
+    align-items: center;
+}
+
+/* disable :hover until Spectrum CSS disables hover upstream */
+
+:host(:hover) {
+    background-color: var(--spectrum-badge-m-textonly-background-color);
+    color: var(--spectrum-badge-m-texticon-text-color);
+}
+
+/* implement fixed placement styling until Spectrum CSS adds it upstream */
```

Comment:
Is there an issue we can track here?

Reply:
Good idea: https://github.com/adobe/spectrum-css/issues/1443